main.go

```diff
@@ -13,9 +14,17 @@ import (
 
 func main() {
 	if err := cmd.Run(context.Background(), os.Args[1:]...); err != nil {
-		if !errors.Is(err, terminal.InterruptErr) {
-			logging.S.Error(err.Error())
+		var userErr cmd.Error
+		switch {
+		case errors.Is(err, terminal.InterruptErr):
+			logging.S.Debug("user interrupted (kill, stop) the process")
+		case errors.As(err, &userErr):
+			fmt.Fprintln(os.Stderr, userErr.Error())
+		default:
+			userErr.OriginalError = err
+			fmt.Fprintln(os.Stderr, userErr.Error())
```

Comment:
```suggestion
			fmt.Fprintf(os.Stderr, "Internal error:\n%v\n", err)
```

(or however we want to format these)

Reply:
done